addons/mail/static/src/models/attachment/attachment_tests.js

```diff
@@ -137,6 +137,28 @@ QUnit.test('isViewable', async function (assert) {
     assert.ok(attachment.isViewable);
 });
 
+QUnit.test('uploadController', async function (assert) {
+    assert.expect(7);
+
+    await this.start();
+    assert.notOk(this.env.models['mail.attachment'].find(attachment => attachment.id === 750));
+
+    const attachment = this.env.models['mail.attachment'].create({
+        filename: "test.txt",
+        id: 750,
+        isTemporary: true,
```

Comment:
put comment next to it that it is what qualifies this attachment as "uploading" (plan to refactor name at a later date)